scripts/verdaccio.sh

```diff
@@ -18,7 +18,6 @@ function cleanup {
   then
     lsof -ti tcp:4000 | xargs kill
     # Clean up generated dists if run locally
-    shopt -s globstar extglob
```

Comment:
can't delete this for the people not on M1
we should really put a conditional around it, but not sure what that would be